.idea/inspectionProfiles/Project_Default.xml

```diff
@@ -0,0 +1,61 @@
+<component name="InspectionProjectProfileManager">
```

Comment:
Yes it was. It should be in .gitignore. Actually it it is.
Unfortunatly this week, it's me, that doesn't have much time to fix things. 
I will try to fix async, so please keep on commenting if you'll find things, that are not ok.